apps/backup/veeam/local/mode/tapejobs.pm

```diff
@@ -161,6 +161,11 @@ sub manage_selection {
             $self->{output}->output_add(long_msg => "skipping job '" . $job->{name} . "': no matching filter type.", debug => 1);
             next;
         }
+        # Sometimes we may get such JSON: [{"lastResult":null,"name":null,"lastState":null,"type":null,"enabled":null}]
+        if (!defined($job->{name}) || $job->{name} eq 'null' ) {
```

Comment:
null is not defined. you can remove the eq 'null' condition

Reply:
belt and braces! 😹 

Indeed ... thanks